src/main/java/au/gov/geoscience/portal/server/services/filters/PetroleumTenementFilter.java

```diff
@@ -33,16 +33,15 @@ public PetroleumTenementFilter(String name, String holder) {
      * @param tenementTypeUris - type of tenement
      */
     public PetroleumTenementFilter(String name, String holder, Set<String> statusUris, Set<String> tenementTypeUris) {
-        PetroleumTenementServiceProviderType petroleumTenementServiceProviderType = PetroleumTenementServiceProviderType.GeoServer;
```

Comment:
There are some imports that are unused now.

Reply:
Removed